lib/auth/init.go

```diff
@@ -1164,6 +1165,7 @@ func encryptCertAuthority(asrv *Server, item backend.Item) error {
 		return trace.Wrap(err)
 	}
 
+	log.Infof("decrypting the cert authority: %v", string(newItem.Key))
```

Comment:
is this the actual key? Let's not print the key decrypted in the logs

Reply:
Its name of the key, not the actual key